tests/reflection/attribute.slang.expected

```diff
@@ -0,0 +1,106 @@
+result code = 0
+standard error = {
+}
+standard output = {
+{
+    "parameters": [
+        {
+            "name": "param",
+            "binding": {"kind": "constantBuffer", "index": 0},
+            "type": {
+                "kind": "parameterBlock",
+                "elementType": {
+                    "kind": "struct",
+                    "name": "A",
+                    "fields": [
+                        {
+                            "name": "x",
+                            "type": {
+                                "kind": "scalar",
+                                "scalarType": "float32"
+                            },
+                            "binding": {"kind": "uniform", "offset": 0, "size": 4}
+                        },
+                        {
+                            "name": "y",
+                            "type": {
+                                "kind": "scalar",
+                                "scalarType": "float32"
+                            },
+                            "binding": {"kind": "uniform", "offset": 4, "size": 4},
+                            "userAttribs": [{
+                                "name": "DefaultValue",
+                                "arg0": 1}
+                            ]
+                        }
+                    ],
+                    "userAttribs": [{
+                        "name": "MyStruct",
+                        "arg0": 0,
```

Comment:
Wouldn't an `arguments` array be a more canonical way to encode this?

```
"arguments": [
    0,
    1.000000
]
```

Reply:
Fixed.